src/lambdas/initiateAthenaQuery/initiateQuery.test.ts

```diff
@@ -53,31 +39,24 @@ describe('initiateQuery', () => {
     mockGetDatabaseEntryByZendeskId.mockResolvedValue({
       requestInfo: testDataRequest
     })
-    jest.spyOn(logger, 'info')
   })
 
-  it('confirms whether the athena data source exists and whether query sql has been generated', async () => {
-    mockConfirmAthenaTable.mockResolvedValue({
-      tableAvailable: true,
-      message: 'test message'
-    })
-    mockGetDatabaseEntryByZendeskId.mockResolvedValue({
-      requestInfo: dataPathsTestDataRequest
-    })
+  it('updates the database when the athena query is initiated', async () => {
     mockCreateQuerySql.mockReturnValue({
       sqlGenerated: true,
       sql: 'test sql string',
       queryParameters: ['123']
     })
-    mockUpdateQueryByZendeskId.mockResolvedValue('test db return object')
-    mockStartQueryExecution.mockResolvedValue({
+    mockStartQueryExecution.mockReturnValue({
       queryExecuted: true,
-      queryExecutionId: 'test_id'
+      queryExecutionId: '123'
```

Comment:
Think I'm going to add this as a test constant, along with the query parameters string